### PR TITLE
feat: table component

### PR DESCRIPTION
**Pull request by scar055**
adds table component to storybook

### CODE REVIEW

**Comment by Robbert** [comment]
@scar055 You don't need to include each stories from the Utrecht Storybook in here, because many of those examples are experimental. Stick to the tables you find in Storybook, and make sure the design tokens match the design.

**Comment by scar055** [comment]
the error import commit can be reverted but it breaks the run storybook command

File: packages/components-css/table/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<!-- @license CC0-1.0 -->
+
+# Example Button component
```
Comment (bddjong) [inline packages/components-css/table/README.md:3]: Table component*

File: packages/components-css/table/index.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Community for NL Design System
+ */
```
Comment (bddjong) [inline packages/components-css/table/index.scss:4]:
```suggestion
/**
 * @license EUPL-1.2
 * Copyright (c) 2023 Rotterdam
 * Copyright (c) 2023 Frameless B.V.
 */
```

File: packages/storybook/config/preview.tsx [hunk 3]
```diff
@@ -7,8 +6,6 @@ import { FigmaBlock } from './FigmaBlock';
 import '@gemeente-rotterdam/design-tokens/dist/index.css';
 import '@gemeente-rotterdam/font/src/index.scss';
 
-defineCustomElements();
```
Comment (bddjong) [inline packages/storybook/config/preview.tsx:10]: Curious why are we deleting this?

Comment (Robbert) [reply packages/storybook/config/preview.tsx:10]: Yup, dit moet blijven staan, anders gaan de icons kapot.

File: packages/components-css/table/index.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+.rods-table-header-cell {
```
[the remark this answers is not in the document]
Comment (Robbert) [reply packages/components-css/table/index.scss]: @scar055 Je hebt het nu wel aangepast in de `.scss`, maar niet in de `className` — dus nu is het kapot.

**Review by Robbert — APPROVED** [review]

**Pull request merged**